packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.doc.tsx

```diff
@@ -2,12 +2,13 @@ import * as React from 'react';
 import { CheckboxBasicExample } from './examples/Checkbox.Basic.Example';
 
 import { IDocPageProps } from '../../common/DocPage.types';
-import { CheckboxImplementationExamples } from './examples/Checkbox.ImplementationExamples';
+import { CheckboxImplementationExamples } from './examples/Checkbox.Other.Example';
```

Comment:
I renamed Checkbox.ImplementationExamples.tsx to Checkbox.Other.Example.tsx to better match other example filenames.